packages/nextjs/test/config/webpack/constructWebpackConfig.test.ts

```diff
@@ -138,7 +139,7 @@ describe('constructWebpackConfigFunction()', () => {
       );
     });
 
-    it('injects user config file into `_app` in client bundle but not in server bundle', async () => {
+    it('injects user config file into `_app` in server bundle but not in client bundle', async () => {
```

Comment:
why did this condition flip?

Reply:
It basically doesn't matter where we inject what. To clean things up a bit I decided we should inject the server SDK in the `pages/_app` entrypoint and the client SDK to the `main` and `app-main` entrypoints.

I assume this test just verified that we do not accidentally mess this up.